tests/end2end/data/hints/short_dict.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+
+<!-- target: hello.txt -->
```

Comment:
You can remove that comment - it's only needed for the files in `data/hints/html` which are handled automatically (where `test_hints` verifies the hint goes to the target given in the comment).